go-vulndb/govulndb.go

```diff
@@ -83,10 +83,12 @@ func (c VulnDB) Update() error {
 		return xerrors.Errorf("index error: %w", err)
 	}
 
+	var modulesErrors []error
 	for moduleName := range modules {
 		entries, err := c.parseModuleEntries(baseURL, moduleName)
 		if err != nil {
-			return xerrors.Errorf("module entry error: %w", err)
+			modulesErrors = append(modulesErrors, err)
```

Comment:
I think we should handle the more specific error. For example, we may want to accept a 404 error here.
https://github.com/aquasecurity/vuln-list-update/blob/c61f90b2481d6769d46db652fb9ecd667e060db1/go-vulndb/govulndb.go#L137

Reply:
Done